packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -104,7 +104,12 @@ export class Choice extends React.PureComponent {
     if (inputProps.onChange) delete inputProps.onChange;
 
     return (
-      <div className={className}>
+      <div
+        className={className}
+        aria-live={checkedChildren ? 'polite' : null}
```

Comment:
These are working well on the following combinations:

* Win10 + IE11 + JAWS
* Win10 + IE11 + Chrome
* Win10 + Firefox + NVDA
* MacOS Mojave + Safari + VoiceOver

It's not announcing properly on iOS + Safari + VoiceOver. That's not necessarily a failure point, but something to be aware of.